src/sap.ui.core/src/sap/ui/model/CompositeBinding.js

```diff
@@ -53,14 +57,12 @@ sap.ui.define(['jquery.sap.global', './BindingMode', './ChangeReason', './Proper
 	};
 	
 	CompositeBinding.prototype.isResolved = function() {
-		var bResolved = false;
 		jQuery.each(this.aBindings, function(i, oBinding) {
-			bResolved = oBinding.isResolved();
-			if (!bResolved) {
+			if (!oBinding.isResolved()) {
 				return false;
 			}
 		});
-		return bResolved;
+		return true;
```

Comment:
This function now always return true, doesn't it? The inner return just breaks the jQuery.each?! By the way, I recommend ES5-style Array.prototype.forEach().